dataikuapi/dss/plugin.py

```diff
@@ -147,4 +175,148 @@ def put_file(self, path, f):
         data = f.read() # eat it all, because making it work with a path variable and a MultifilePart in swing looks complicated
         return self.client._perform_empty("POST", "/plugins/%s/contents/%s" % (self.plugin_id, path), raw_body=data)
 
-        
\ No newline at end of file
+
+class DSSPluginUsage(object):
+    """
+    Information on a usage of an element of a plugin.
+    """
+    def __init__(self, json_object):
+        self._json_object = json_object
+
+    def get_raw(self):
+        return self._json_object
+
+    @property
+    def element_kind(self):
+        """
+        Element kind (webapps, python-formats,...)
+        :return: the element kind
+        :rtype: str
+        """
+        return self._json_object["elementKind"]
+
+    @property
+    def element_type(self):
+        """
+        Element type
+        :return: the element type
+        :rtype: str
+        """
+        return self._json_object["elementType"]
```

Comment:
Here and above you do not use `.get()`. So you expect these field to always exist ? If not you you might want to catch the KeyError exception that can be raised and re-raise it with a more comprehensive message for the API user ?